docs/08-operations/ci-cd/coderabbit-quick-reference.md

```
@@ -0,0 +1,272 @@
+
+**For:** Developers creating PRs
+**Last Updated:** November 15, 2025
+
+---
+
+## 🚀 Quick Commands
+
+### In PR Comments
+
+| Command | What It Does |
+|---------|-------------|
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)